.trybe/requirements.json

```
@@ -1,51 +1,51 @@
 {
   "requirements": [
     {
-      "description": "A página deve possuir o título \"Paleta de Cores\".",
+      "description": "1 - Adicione à página o título \"Paleta de Cores\".",
       "bonus": false
     },
     {
-      "description": "A página deve possuir uma paleta de quatro cores distintas.",
+      "description": "2 - Adicione à página uma paleta de quatro cores distintas.",
       "bonus": false
     },
     {
-      "description": "A cor preta deve ser a primeira na paleta de cores.",
+      "description": "3 - Adicione na página a cor **preta** ela deve ser a primeira na paleta de cores.",
       "bonus": false
     },
     {
-      "description": "A página deve possuir um quadro de pixels, com 25 pixels.",
+      "description": "4 - Adicione à página um quadro de pixels, com 25 pixels.",
       "bonus": false
     },
     {
-      "description": "Cada elemento do quadro de pixels deve possuir 40 pixels de largura e 40 pixels de altura e ser delimitado por uma borda preta de 1 pixel.",
+      "description": "5 - Faça com que cada elemento do quadro de pixels possua 40 pixels de largura, 40 pixels de altura e seja delimitado por uma borda preta de 1 pixel.",
       "bonus": false
     },
     {
-      "description": "Ao carregar a página, a cor preta da paleta já deve estar selecionada para pintar os pixels.",
+      "description": "6 - Definia a cor preta como cor inicial. Ao carregar a página a cor preta já deve estar selecionada para pintar os pixels",
       "bonus": false
     },
     {
-      "description": "Ao clicar em uma das cores da paleta, a cor selecionada é que vai ser usada para preencher os pixels no quadro.",
+      "description": "7 -  Selecione uma das cores da paleta, ao clicar, a cor selecionada é a que será utilizada para preencher os pixels no quadro.",
```

O que acha dessa alteração?
```suggestion
      "description": "7 -  Clicar em uma das cores da paleta, faz com que ela seja selecionada e utilizada para preencher os pixels no quadro.",
```